### PR TITLE
Correction of some errors in the documentation

### PR DESCRIPTION
**Pull request by moresdavidewayan**

### CODE REVIEW

**Comment by kearfy** [comment]
Hey @moresdavidewayan, could you rebase this? #91 has since been fixed in #106

File: app/templates/docs/installation/running/file.hbs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 	<h2>Run a single-node, on-disk server</h2>
 
-	<p>For the purposes of getting started with SurrealDB quickly, we will start an in-memory database which does not persist data on shutdown. This database is great for development and testing.</p>
+	<p>For the purposes of getting started with SurrealDB quickly, we will start an in-memory database which persists data on shutdown. This database is great for development and testing.</p>
```
Comment (phughk) [inline app/templates/docs/installation/running/file.hbs:11]:
```suggestion
	<p>For the purposes of getting started with SurrealDB quickly, we will start a RocksDB database which persists data on the filesystem. This configuration is great for trying out the database and small deployments.</p>
```

File: app/templates/docs/installation/running/file.hbs [hunk 2]
```diff
@@ -29,7 +29,7 @@
 	<p>In order to change the default port that SurrealDB uses for web connections and from database clients you can use the <Ascua::Prism::Inline @language="bash" @code="--bind" /> argument. The following command starts the database on port <code>8080</code>.</p>
 
 	<Code @name="docs-installation-running-file-4.shell">
-		user@localhost % surreal start --log trace --user root --pass root --bind 0.0.0.0:8080 memory
+		user@localhost % surreal start --log trace --user root --pass root --bind 0.0.0.0:8080 file:mydatabase.db
```
Comment (phughk) [inline app/templates/docs/installation/running/file.hbs:32]:
```suggestion
		user@localhost % surreal start --log trace --user root --pass root --bind 0.0.0.0:8080 file://path/to/mydatabase
```

File: app/templates/docs/installation/running/tikv.hbs [hunk 3]
```diff
@@ -56,7 +56,7 @@
 	<p>In order to change the default port that SurrealDB uses for web connections and from database clients you can use the <Ascua::Prism::Inline @language="bash" @code="--bind" /> argument. The following command starts the database on port <code>8080</code>.</p>
 
 	<Code @name="docs-installation-running-tikv-6.shell">
-		user@localhost % surreal start --log trace --user root --pass root --bind 0.0.0.0:8080 memory
+		user@localhost % surreal start --log trace --user root --pass root --bind 0.0.0.0:8080 tikv.//127.0.0.1:2379
```
Comment (phughk) [inline app/templates/docs/installation/running/tikv.hbs:59]:
```suggestion
		user@localhost % surreal start --log trace --user root --pass root --bind 0.0.0.0:8080 tikv://placement.driver.hostname.or.ip:2379
```

**Review by phughk — COMMENTED** [review]
Some suggestions. Thank you so much for finding and correcting these @moresdavidewayan !

**Comment by moresdavidewayan** [comment]
Ok, I'll make the changes

**Pull request merged**